WIP

JOwen-ster opened the pull request.

JOwen-ster committed:
Update .env

JOwen-ster committed:
gitignore

JOwen-ster committed:
Delete .env

k4rni reviewed (approved):
Thank you for helping out on this project! :godmode:

JOwen-ster committed:
Update santa_bot.py

ADD - Listing participants, clearing list, adding, removing, Nickname listener for list

Update - Modal

JOwen-ster committed:
gitignore

JOwen-ster committed:
Update gitignore

JOwen-ster committed:
Merge branch 'main' of https://github.com/JOwen-ster/Secret-Santa-Bot

JOwen-ster committed:
gitignore
